webtranslate/data.py

```diff
@@ -95,28 +106,38 @@ class StatusDefinition:
     @ivar baselng: Whether the status exists for base languages.
     @type baselng: C{bool}
     """
+
     def __init__(self, code, name, description, baselng):
         self.code = code
         self.name = name
         self.description = description
         self.baselng = baselng
 
 
-MISSING_OK =  0 # String is missing, but that's allowed (non-default case string).
-UNKNOWN =     1 # String has unknown state.
-UP_TO_DATE =  2 # String is newer than the base string.
-OUT_OF_DATE = 3 # String is older than the base string.
-INVALID =     4 # String is invalid relative to the base string.
-MISSING =     5 # The default case string is missing.
-
-MAX_STATE =   6
-
-STATUSES = [StatusDefinition(MISSING_OK,  'Omitted',  'The case was not needed in the translation',                                                 True),
-            StatusDefinition(UNKNOWN,     'Unknown',  'The state of the translation was not decidable',                                             False),
-            StatusDefinition(UP_TO_DATE,  'Correct',  'The string is technically correct and up to date',                                           True),
-            StatusDefinition(OUT_OF_DATE, 'Outdated', 'A valid translation exists, but it needs review as a newer base language text is available', False),
-            StatusDefinition(INVALID,     'Invalid',  'A translation exists, but its string parameters do not match with the base language',        True),
-            StatusDefinition(MISSING,     'Missing',  'No translation could be found',                                                              False),]
+MISSING_OK = 0  # String is missing, but that's allowed (non-default case string).
+UNKNOWN = 1  # String has unknown state.
+UP_TO_DATE = 2  # String is newer than the base string.
+OUT_OF_DATE = 3  # String is older than the base string.
+INVALID = 4  # String is invalid relative to the base string.
+MISSING = 5  # The default case string is missing.
+
+MAX_STATE = 6
+
+STATUSES = [
```

Comment:
might want to consider a `fmt: off` block around this table (with the improved trailing `[]` ofc)

Reply:
I think the current new situation is a huge improvement of the old. Using spaces to align arbitrary lists is a stupid and silly thing humans tend to do. For any textual change, you have to change all the entries around it. I am heavily in favour of the new, not of the old :P So I kinda don't want to put `fmt` off :D